docs/Displaying a User Interface/Alerts/index.md

```diff
@@ -224,6 +224,37 @@ alert.setPlayTone(true);
 ```
 !@
 
+### Alert Icon
+
+###### Alert With Non Default Icon
+![Generic - Alert](assets/Generic_alertIcon.png)
+
+An alert can include a custom or static image that will be displayed within the alert. Before you add the image to the alert make sure the image is uploaded to the head unit. If you are certain the image is already uploaded, you can just set `alertIcon` right away.
+
+@![iOS]
+##### Objective-C
+```objc
+[self.sdlManager.fileManager uploadArtwork:[SDLArtwork artworkWithImage:[UIImage imageNamed:<#ImageName#>] asImageFormat:SDLArtworkImageFormatPNG] completionHandler:^(BOOL success, NSString * _Nonnull artworkName, NSUInteger bytesAvailable, NSError * _Nullable error) {
```

Comment:
I think this example might be misleading. If they put this inline with the rest of the example code, it will fail because it's async. Remove the upload code and just include the `alert.alertIcon = [[SDLImage alloc] initWithName:artworkName isTemplate:YES];` code.